atd-vzv/src/views/summary/CrashesByMode.js

```diff
@@ -239,160 +240,170 @@ const CrashesByMode = () => {
       </Row>
       <Row>
         <Col>
-          <hr className="mb-2" />
+          <hr />
         </Col>
       </Row>
-      <Row className="mt-1">
-        <Col>
-          {chartLegend}
-          {
-            <Container>
-              <Bar
-                ref={(ref) => (chartRef.current = ref)}
-                data={data}
-                height={null}
-                width={null}
-                options={{
-                  responsive: true,
-                  aspectRatio: 1.37,
-                  maintainAspectRatio: false,
-                  scales: {
-                    xAxes: [
-                      {
-                        stacked: true,
-                      },
-                    ],
-                    yAxes: [
-                      {
-                        stacked: true,
-                      },
-                    ],
-                  },
-                  legend: {
-                    display: false,
-                  },
-                  legendCallback: function (chart) {
-                    return (
-                      <Row className="pb-2">
-                        <Col className="pr-1 col-sm-4">
-                          <StyledDiv>
-                            <div>
-                              <p
-                                className="h6 text-center my-1 pt-2"
-                                style={{ height: "27px" }}
-                              ></p>
-                            </div>
-                            {chart.data.datasets.map((dataset, i) => {
-                              const updateLegendColors = () => {
-                                const legendColorsClone = [...legendColors];
-                                legendColors[i] !== "dimgray"
-                                  ? legendColorsClone.splice(i, 1, "dimgray")
-                                  : legendColorsClone.splice(
-                                      i,
-                                      1,
-                                      chartColors[i]
-                                    );
-                                setLegendColors(legendColorsClone);
-                              };
+      {!!data.datasets ? (
```

Comment:
Way out of scope, but it is interesting to think about how we could use these spinners as a fallback for when the ETL fails every once in a blue moon and the API returns an empty array of data records. We'll probably have a chance to avoid that problem altogether with the new data model, though! 🤞